mpiis_doc_build/resources/sphinx/conf.py.in

```diff
@@ -0,0 +1,245 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import datetime
+import os
+import sys
+from m2r import MdInclude
```

Comment:
m2r is for allowing a mix of Markdown and rst, right?  I anyway found this possible mixing of two styles in one file a bit confusing, so I'm happy to drop it.

I think the officially supported way of using Markdown with Sphinx is using the [MyST parser](https://myst-parser.readthedocs.io/en/latest/intro.html), so I would suggest switching to that.

Pro:
- It is the official suggestion in the [Sphinx documentation](https://www.sphinx-doc.org/en/master/usage/markdown.html).
- Clearer separation between Markdown and rst (you can use both md and rst files but within one file you have to be consistent).
- MyST has some special syntax (non-standard markdown) to support rst roles/directives, so you still have all the options to use references, add autodoc entries, etc. when using Markdown.

Con:
- It would be a breaking change as a mix-up of both markdown and rst within one file would not work anymore.  However, my opinion is that this should not be done anyway, so I would be okay with this.

Reply:
I created an issue for this: #2